templates/core/index.html

```diff
@@ -33,4 +33,16 @@ <h2>Recommended venues:</h2>
 		{% endfor %}
 	</ul>
 </section>
+<section class="hero_component_container">
+	<div class="hero_info">
+		<h3>The best creative spaces, cafes for a quiet spot away from work</h3>
+		<p>Find a co-working spot Lorem ipsum dolor sit amet, consectetur adipiscing elit. Maecenas odio augue, sollicitudin sed dui vitae, vulputate auctor nisl. Morbi mollis at mi quis tincidunt. Proin ac nisl congue, pretium mi et, convallis nunc. Aliquam ac posuere eros, eget interdum nisi. Integer a elementum nisl, sed luctus nisl. Maecenas quis lacus fringilla, condimentum urna vitae, facilisis neque. Proin semper aliquam metus, eget imperdiet mi. Fusce consequat ac diam at accumsan.</p>
+		<button class="hero">
+			<a href="{% url 'about' %}">About us</a>
+		</button>
+	</div>
+	<div class="hero_image">
+		<img class="menu_image" src="https://trello-attachments.s3.amazonaws.com/5b5deb66b4e3542422dbb2fd/5cc619deb2236e1f7be917f5/0d599e4dd34fa26912b2a1084adf7904/menu_(1).svg" alt="menu">
```

Comment:
Please have a read https://docs.djangoproject.com/en/3.0/howto/static-files/

- We can create a folder `src/img` and add images there (nicely named!)
- add some changes to webpack to copy the files from `src/img` to `dist/img` look at webpack Copy tasks
and that will copy our images from `src` straight to static folder